src/richie/apps/courses/templates/courses/cms/blogpost_detail.html

```diff
@@ -32,97 +34,123 @@ <h1 class="subheader__title">{% page_attribute "page_title" request.current_page
+                    </div>
+                    {% endif %}
+                {% endwith %}
+            {% endblock authors %}
         </div>
+        {% endblock subhead %}
 
-        <h1 class="blogpost-detail__title">{% render_model current_page "title" %}</h1>
+        {% block title %}
+            <h1 class="blogpost-detail__title">{% render_model current_page "title" %}</h1>
+        {% endblock title %}
 
-        {% include "social-networks/blogpost-badges.html" with page_title=request.current_page.get_title page_url=request.current_page.get_absolute_url %}
+        {% block social_networks %}
+            {% include "social-networks/blogpost-badges.html" with page_title=request.current_page.get_title page_url=request.current_page.get_absolute_url %}
+        {% endblock social_networks %}
 
         <div class="blogpost-detail__grid">
+            {% block grid_main %}
             <div class="blogpost-detail__grid-main">
 
-                <div class="blogpost-detail__cover">
-                    {% get_placeholder_plugins "cover" as plugins or %}
-                        <p class="blogpost-detail__empty">{% trans "Cover" %}</p>
-                    {% endget_placeholder_plugins %}
-                    {% blockplugin plugins.0 %}
-                        <img
-                            src="{% thumbnail instance.picture 845x500 subject_location=instance.picture.subject_location %}"
-                            srcset="
-                                {% thumbnail instance.picture 845x500 subject_location=instance.picture.subject_location %} 500w
-                                {% if instance.picture.width >= 1000 %},{% thumbnail instance.picture 1000x1000 subject_location=instance.picture.subject_location %} 1000w{% endif %}
-                                {% if instance.picture.width >= 2000 %},{% thumbnail instance.picture 2000x2000 subject_location=instance.picture.subject_location %} 2000w{% endif %}
-                            "
-                            sizes="500px"
-                            alt="{% if instance.picture.default_alt_text %}{{ instance.picture.default_alt_text }}{% else %}{% trans 'blog post cover image' %}{% endif %}"
-                        />
-                    {% endblockplugin %}
-                </div>
-
-                <div class="blogpost-detail__excerpt">
-                    {% placeholder "excerpt" or %}
-                        <p class="blogpost-detail__empty">
-                            {% trans "No excerpt content" %}
-                        </p>
-                    {% endplaceholder %}
-                </div>
-
-                <div class="blogpost-detail__body">
-                    {% placeholder "body" or %}
-                        <p class="blogpost-detail__empty">
-                            {% trans "No body content" %}
-                        </p>
-                    {% endplaceholder %}
-                </div>
+                {% block cover %}
```

Comment:
This is very much a nitpick but you should probably either always indent or never indent after a block declaration.

Given 4-space indentation is already a little bit painful to work with wrt. excessive indentation all around, I'd lean towards no indentation after block declarations 😇

Reply:
Done! The diff is now unreadable ^^